packages/design-tokens/src/tokens/functional/components/prose/colors.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  brand: {
+    Prose: {
+      listStyleImageUrl: {
```

Comment:
Would like some input as to what a good name for this value should be. Any thoughts @rezrah ?

Reply:
How about `--brand-Prose-unorderedList-item-bgUrl`

Reply:
`bg` I don't think is quite right, as it isn't a background.

Reply:
Fair, `imageUrl` should be fine